app/assets/stylesheets/components/_stuff.scss

```diff
@@ -604,5 +604,91 @@
 
 // Hidden Class
 .d-hide{
-  display:none;
+  display: none;
+}
+
+// Custom swatches
+.variant-selection {
+  .selected-value {
+    font-weight: $font-weight-bold;
+  }
+}
+.selection-container {
+  display: flex;
+  flex-direction: row;
+  margin: 0 -1rem;
+
+  .selection-item {
+    padding: 1rem;
+    text-align: center;
+    position: relative;
+
+    input[type="radio"] {
+      // opacity: 0;
+      position: absolute;
+      // height: 0;
+      // width: 0;
+
```

Comment:
If I try to set the `opacity` property or the sizes, or if I add the `display: none` property (or every rule that hide the input), these tests fail:
https://app.circleci.com/pipelines/github/solidusio/solidus-demo/98/workflows/d9a4a404-a230-4540-bbf3-f9bd9ddfeb67/jobs/107